app/views/users/phone_setup/index.html.erb

```diff
@@ -37,20 +44,8 @@
       ).with_content(t('forms.buttons.send_one_time_code')) %>
 <% end %>
 
-<p class="margin-top-5">
-  <%= t('two_factor_authentication.phone_fee_disclosure') %>
-  <% if IdentityConfig.store.voip_block %>
-    <%= t('two_factor_authentication.two_factor_choice_options.phone_info_no_voip') %>
-  <% end %>
-</p>
-<% unless FeatureManagement.phone_recaptcha_enabled? %>
-  <%= link_to t('two_factor_authentication.mobile_terms_of_service'),
-              MarketingSite.messaging_practices_url,
-              class: 'display-block margin-bottom-4' %>
-<% end %>
-
-<%= render 'shared/cancel_or_back_to_options' %>
-
 <% if FeatureManagement.phone_recaptcha_enabled? %>
   <%= render 'shared/recaptcha_disclosure' %>
```

Comment:
With this no longer being at the bottom of the page, I think we could revisit the margins on the partial:

https://github.com/18F/identity-idp/blob/ac2e6d397b2566b2e192af8505eefbc97e7c63c6/app/views/shared/_recaptcha_disclosure.html.erb#L1

Specifically wondering if we could remove those utility classes, and rely on margins from the submit button (2.5rem bottom) and footer section (2rem top) instead.

I'm also not sure it makes a lot of sense to be a partial anymore as well, now that we have a single page for phone setups. We could inline it here.

Reply:
Definitely agree on removing the partial and moving the disclosure to be inline.

We can remove the `margin-bottom-0` for the class with no adverse effect. The send code button does not naturally have margin on it, so we would still have to keep the `margin-top-4`

![image](https://github.com/18F/identity-idp/assets/17969963/346cc4f1-ce88-4369-a6da-67dda33b180d)

Reply:
> The send code button does not naturally have margin on it, so we would still have to keep the `margin-top-4`

Good point. I'd wonder if we'd want the margin on that instead. Doesn't make much of a difference either way. I'd expect the unit should probably change from `4` to `5` at least, for the standard vertical margins on primary buttons.